modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_0_0/PollsUpgradeProcess.java

```diff
@@ -0,0 +1,1074 @@
+			DDMFormFieldTypeConstants.RADIO);
+
+		ddmFormField.setDataType("string");
+		ddmFormField.setDDMFormFieldOptions(ddmFormFieldOptions);
+		ddmFormField.setLabel(
+			_getLocalizedValue(_defaultLocale, ddmFormFieldLabel));
+		ddmFormField.setLocalizable(true);
+		ddmFormField.setProperty("inline", false);
+		ddmFormField.setProperty("instanceId", StringUtil.randomString(8));
+		ddmFormField.setProperty("visible", true);
+		ddmFormField.setRequired(true);
+		ddmFormField.setShowLabel(false);
+
+		return ddmFormField;
+	}
+
+	protected String getDDMFormFieldName(String ddmFormFieldName) {
+		for (int i = 0; i < 8; i++) {
+			ddmFormFieldName = ddmFormFieldName.concat(
+				String.valueOf(RandomUtil.nextInt(10)));
+		}
+
+		return StringUtil.removeChar(ddmFormFieldName, CharPool.SPACE);
+	}
+
+	protected String getDDMFormLayoutDefinition(DDMFormField ddmFormField) {
+		DDMFormLayout ddmFormLayout = new DDMFormLayout();
+
+		ddmFormLayout.addDDMFormLayoutPage(_getDDMFormLayoutPage(ddmFormField));
+		ddmFormLayout.setDefaultLocale(_defaultLocale);
+		ddmFormLayout.setPaginationMode(DDMFormLayout.MULTI_PAGES);
+
+		DDMFormLayoutSerializerSerializeResponse
+			ddmFormLayoutSerializerSerializeResponse =
+				_ddmFormLayoutSerializer.serialize(
+					DDMFormLayoutSerializerSerializeRequest.Builder.newBuilder(
+						ddmFormLayout
+					).build());
+
+		return ddmFormLayoutSerializerSerializeResponse.getContent();
+	}
+
+	protected String getSerializedSettingsDDMFormValues() {
+		DDMForm ddmForm = DDMFormFactory.create(DDMFormInstanceSettings.class);
+
+		DDMFormValues ddmFormValues = new DDMFormValues(ddmForm);
+
+		ddmFormValues.setAvailableLocales(ddmForm.getAvailableLocales());
+		ddmFormValues.setDefaultLocale(ddmForm.getDefaultLocale());
+
+		for (DDMFormField ddmFormField : ddmForm.getDDMFormFields()) {
+			DDMFormFieldValue ddmFormFieldValue = new DDMFormFieldValue();
+
+			ddmFormFieldValue.setFieldReference(
+				ddmFormField.getFieldReference());
+			ddmFormFieldValue.setName(ddmFormField.getName());
+			ddmFormFieldValue.setValue(
+				_getSettingDefaultValue(ddmForm, ddmFormField));
+
+			ddmFormValues.addDDMFormFieldValue(ddmFormFieldValue);
+		}
+
+		return DDMFormValuesSerializeUtil.serialize(
+			ddmFormValues, _ddmFormValuesSerializer);
+	}
+
+	private long _addDDMField(
+			long companyId, long storageId, long structureVersionId,
+			String fieldName, String fieldType, String instanceId,
+			boolean localizable, int priority)
+		throws Exception {
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				StringBundler.concat(
+					"insert into DDMField (mvccVersion, ctCollectionId, ",
+					"fieldId, companyId, parentFieldId, storageId, ",
+					"structureVersionId, fieldName, fieldType, instanceId, ",
+					"localizable, priority) values (0, 0, ?, ?, ?, ?, ?, ?, ",
+					"?, ?, ?, ?)"))) {
+
+			long fieldId = increment(DDMField.class.getName());
+
+			preparedStatement.setLong(1, fieldId);
+
```

Comment:
@natocesarrego  I think it's necessary because I'm initializing the variable `long fieldId = increment();` above and the`preparedStatement.setLong(1, fieldId);` depends on this variable.

I was also following the example from [DLFileEntryTypeDataDefinitionIdUpgradeProcess.java#L223-L235](https://github.com/liferay/liferay-portal/blob/08c6b85908e5745c20b53fd5c8e3032bbf15e165/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_0/DLFileEntryTypeDataDefinitionIdUpgradeProcess.java#L223-L235).